streams/src/main/java/org/apache/kafka/streams/StreamsMetrics.java

```diff
@@ -100,6 +155,7 @@ Sensor addThroughputSensor(final String scopeName,
      * @param sensor add Sensor whose throughput we are recording
      * @param value  throughput value
      */
+    @Deprecated
     void recordThroughput(final Sensor sensor,
                           final long value);
 
```

Comment:
KIP-444 does not talk about `addLatencyAndThroughputSensor()` and `addThroughputSensor()`. They contain a subset of the metrics in the newly added user-customizable sensors. Additionally, they also create a parent sensor which they new ones do not. 
Should `addLatencyAndThroughputSensor()` and `addThroughputSensor()` be deprecated or the parent sensor be omitted for built-in metrics version `latest`?

Reply:
Sounds to me like they should be deprecated as the newly added metrics will eventually replace them fully.  But I'm not sure ATM.

Reply:
Thinking about this more considering the overlap of `addLatencyAndThroughputSensor()` and `addThroughputSensor()` with the newly added methods in this PR,  I'm confirming my vote for deprecation.

But it would be good to get the opinion of others on this. 
\cc @guozhangwang

Reply:
My original KIP was to just use the old `addLatencyAndThroughputSensor` and `addThroughputSensor` names but with new semantics (without parent hierarchies), but later we renamed them to `addLatencyRateTotalSensor` and `addRateTotalSensor`. So I think in this way we should also deprecate the former two, will update the KIP also.

Reply:
Right, now I remember. Thank you for updating the KIP.